README.md

```diff
@@ -159,6 +160,26 @@ A simple example on how to do so, the user must deploy a `ConfigMap`, such as in
 
 Currently, this configuration is not dynamic.
 
+## KubeSecondaryDNS
+
+[This controller](https://github.com/kubevirt/kubesecondarydns)
+allows to support FQDN for VMI's secondary networks.
+
+```yaml
+apiVersion: networkaddonsoperator.network.kubevirt.io/v1
+kind: NetworkAddonsConfig
+metadata:
+  name: cluster
+spec:
+  kubeSecondaryDNS:
+    DOMAIN: ""
+    NAME_SERVER_IP: ""
+```
+
+Additionally, container image used to deliver this plugin can be set using
```

Comment:
that's true for all components, why is this important to state here?

Reply:
Saw the macVtap state his as well, so added it here as well 
can remove if needed, it just show what is the parameter that can be used for it

Reply:
I think you mean bridge marker. But the order is different, making it a bit less clear.
may I suggest:
```
The KubeSecondaryDNS controller allows FQDN support for VMI's secondary networks.

The operator allows administrator to deploy [KubeSecondaryDNS plugin](https://github.com/kubevirt/kubesecondarydns) by adding kubeSecondaryDNS attribute to NetworkAddonsConfig.
<<ADD MANIFEST HERE>>
Where DOMAIN is ...and NAME_SERVER_IP is.

Additionally, container image used to deliver this plugin can be set using `KUBE_SECONDARY_DNS_IMAGE` environment variable in operator
deployment manifest.
```

Reply:
So lets keep as is please
(btw meant `MULTUS_DYNAMIC_NETWORKS_CONTROLLER_IMAGE`)